src/status_im/ui/screens/status/views.cljs

```diff
@@ -78,7 +78,8 @@
 (defn on-long-press-fn [on-long-press content image]
   (on-long-press
    (when-not image
-     [{:on-press #(react/copy-to-clipboard
+     [{:id "copy"
```

Comment:
why string ?

Reply:
https://github.com/status-im/status-react/blob/features%2Fsuggest-tip/src/status_im/ui/screens/chat/message/reactions_picker.cljs#L26

<img width=320 src=https://media.giphy.com/media/XeXzWgD6P4LG8/giphy.gif />

Reply:
it works with a keyword though